polaris-core/src/main/java/org/apache/polaris/core/entity/PolarisPrivilege.java

```diff
@@ -155,6 +155,91 @@ public enum PolarisPrivilege {
       PolarisEntityType.POLICY,
       PolarisEntitySubType.NULL_SUBTYPE,
       PolarisEntityType.CATALOG_ROLE),
+  TABLE_ASSIGN_UUID(
```

Comment:
Can we add another privilege, like `TABLE_MANAGE_STRUCTURE` or something similar, that is a super privilege for basically everything except `TABLE_ADD_SNAPSHOT` and `TABLE_SET_SNAPSHOT_REF`? A lot of users will want to grant table-management privileges all at once, rather than having to grant `TABLE_SET_PROPERTIES` and `TABLE_REMOVE_PROPERTIES` separately. Similarly, I think `TABLE_ADD_SNAPSHOT` and `TABLE_SET_SNAPSHOT_REF` could fall under a common parent

Reply:
To me, it's not hard to imagine a table maintenance service that has the ability to expire snapshots or change data retention policies without the ability to read or write table data